AUDIO/BlockSpeechContextGroup.md

```diff
@@ -0,0 +1,30 @@
+---
+ns: AUDIO
+aliases: ["0xA8A7D434AFB4B97B"]
+---
+## BLOCK_SPEECH_CONTEXT_GROUP
+
+```c
+// 0xA8A7D434AFB4B97B
+void BLOCK_SPEECH_CONTEXT_GROUP(char* groupName, int contextBlockTarget);
+```
+
+```c
+enum eAudContextBlockTarget {
+	AUD_CONTEXT_BLOCK_PLAYER = 0,
+	AUD_CONTEXT_BLOCK_NPCS = 1,
+	AUD_CONTEXT_BLOCK_BUDDYS = 2,
+	AUD_CONTEXT_BLOCK_EVERYONE = 3,
+
+	AUD_CONTEXT_BLOCK_TARGETS_TOTAL
+}
+```
```

Comment:
Same here, would be better to switch on a table method.

| Enum Name                   | Value | Description                  |
|-----------------------------|-------|------------------------------|
| AUD_CONTEXT_BLOCK_PLAYER    | 0     | Block context for player     |
| AUD_CONTEXT_BLOCK_NPCS      | 1     | Block context for NPCs       |
| AUD_CONTEXT_BLOCK_BUDDYS    | 2     | Block context for buddies    |
| AUD_CONTEXT_BLOCK_EVERYONE  | 3     | Block context for everyone   |
| AUD_CONTEXT_BLOCK_TARGETS_TOTAL |       | Total number of block targets|

Reply:
Refer to previous response